src/scriptworker/constants.py

```diff
@@ -193,7 +193,7 @@
                                     r"^(?P<path>/mozilla-(central|unified))(/|$)",
                                     r"^(?P<path>/integration/(autoland|fx-team|mozilla-inbound))(/|$)",
                                     r"^(?P<path>/releases/mozilla-(beta|release|esr\d+))(/|$)",
-                                    r"^(?P<path>/projects/(maple|oak|cedar|pine))(/|$)",
```

Comment:
Question: I see `cedar` was last updated 3 months ago[1]. Although `oak` was last modified 9 months ago.  


![Screenshot 2022-06-22 at 10 10 05](https://user-images.githubusercontent.com/5907366/174978398-576af3c2-ce5c-4c18-925e-373412dea30c.png)

Do we want to get rid of `oak` as well?


[1] https://hg.mozilla.org/projects?sort=lastchange

Reply:
https://wiki.mozilla.org/ReleaseEngineering/DisposableProjectRepositories#BOOKING_SCHEDULE has oak booked by the install-update team, might be worth checking with them.

Reply:
Oh TIL about this page 🙂 Yeah, we may want to check with the rest of the engineering org if can close a few of these branches, then.